wcfsetup/install/files/ts/WoltLabSuite/Core/User.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Provides data of the active user.
+ *
+ * @author  Matthias Schmidt
+ * @copyright  2001-2019 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  User (alias)
+ * @module  WoltLabSuite/Core/User
+ */
+
+let _link: string | undefined;
+
+export = {
+  /**
+   * Returns the link to the active user's profile or an empty string
+   * if the active user is a guest.
+   */
+  getLink: (): string => _link || '',
```

Comment:
Why is this not using `export function getLink()`? Is it because of the `defineProperty` on `this`? 

Then please do the following for consistency, please:

```suggestion
  getLink(): string => {
    return _link || '';
  }
```